src/v3/src/components/AuthContainer/AuthContainer.tsx

```diff
@@ -11,13 +11,15 @@
  */
 
 import { Box } from '@mui/material';
+import useMediaQuery from '@mui/material/useMediaQuery';
 import classNames from 'classnames';
 import { FunctionComponent, h } from 'preact';
 
 import style from './style.css';
 
 const AuthContainer: FunctionComponent = ({ children }) => {
   const classes = classNames('auth-container', 'main-container', style.mainViewContainer);
+  const borderMatches = useMediaQuery('screen and (max-width: 391px)');
```

Comment:
nit: can we rename this to reflect that it's checking if it's in mobile? like `isNarrowViewport` or `isMobileWidth`?

Reply:
yep good point